assets/vscode/.vscode/templatec_cpp_properties.json

```diff
@@ -0,0 +1,61 @@
+{
+    "configurations": [
+        {
+            "name": "Win32",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "${config:MAXIM_PATH}/Libraries/PeriphDrivers/Include/${config:target}",
+                "${config:MAXIM_PATH}/Libraries/Boards/${config:target}/Include",
+                "${config:MAXIM_PATH}/Libraries/Boards/${config:target}/${config:board}/Include",
+                "${config:MAXIM_PATH}/Libraries/CMSIS/Device/Maxim/${config:target}/Include",
+                "${config:MAXIM_PATH}/Libraries/CMSIS/Include",
+                "${config:MAXIM_PATH}/Tools/GNUTools/arm-none-eabi/include",
+                "${config:MAXIM_PATH}/Tools/GNUTools/lib/gcc/arm-none-eabi/9.2.1/include",
```

Comment:
Is there a way to make this version independent?

Reply:
I don't see a way to make this version _independent_, but I can add a config option.
`"${config:MAXIM_PATH}/Tools/GNUTools/lib/gcc/arm-none-eabi/${config:libgcc_version}/include"`

The `vscode()` generator can then scan `${config:MAXIM_PATH}/Tools/GNUTools/lib/gcc/arm-none-eabi/` and use the latest version # if multiple are available.

... which brings up the point - do we need to handle `MAXIM_PATH` better?  Right now the default location `C:/MaximSDK` is used.

Reply:
Added preliminary option for changing libgcc version #.  (See [28e8bb0](https://github.com/MaximIntegratedAI/ai8x-synthesis/pull/185/commits/28e8bb087a44f0122157925f9b5f0b7ad061f18e))  Since 9.2.1 is the only option in the SDK, it's hard-coded.  This can be updated in the template file if newer versions are added.  Down the line, the SDK will be auto-searched...  

... speaking of which -  I started writing out a method for auto-detecting the MaximSDK.  The scope of that is growing to where it needs a separate PR.  Its utility can also be applied outside of VSCode, and an `sdk_path` state variable could be added